bin/client/justfile

```diff
@@ -62,15 +62,42 @@ run-client-asterisc block_number l1_rpc l1_beacon_rpc l2_rpc rollup_node_rpc ver
     --data-dir ./data \
     {{verbosity}}
 
+# Run the client program natively with the host program attached, against the op-devnet.
+run-client-native-against-devnet verbosity='' block_number='' rollup_config_path='':
+  #!/usr/bin/env bash
+  L1_RPC="http://127.0.0.1:8545"
+  L1_BEACON_RPC="http://127.0.0.1:5052"
+  L2_RPC="http://127.0.0.1:9545"
+  ROLLUP_NODE_RPC="http://127.0.0.1:7545"
+  ROLLUP_CONFIG_PATH="../../../optimism/.devnet/rollup.json"
+
```

Comment:
rm ./data

Reply:
no trying to upstream this to kona. They prob want it there because they use it for something.